index.js

```diff
@@ -25,6 +24,10 @@ async function zeroEks (args) {
     args.pathToNodeBinary = pathTo('node')
   }
 
+  if (checkForTranspiledCode(args.argv[0])) {
+    console.warn('Transpiled code is not supported')
```

Comment:
Should we use a similar message to Bubbleprof? 'The code is transpiled, 0x does not support source maps yet.'

I think this check should happen a bit later—right now it happens before any argument validation, and if your CLI arguments are wrong that is a more important issue to deal with. If you're using the `--visualize-only` flag, the input filename may not exist (or may not be provided in the first place).

I would probably put it just before the call to `startProcessAndCollectTraceData`, when we know for sure that we're actually starting a subprocess.